src/components/molecules/ScheduleItemNG/ScheduleItemNG.tsx

```diff
@@ -99,6 +101,16 @@ export const ScheduleItemNG: React.FC<ScheduleItemNGProps> = ({
     }
   }, [enterRoom, event, eventRoom, roomUrlParam]);
 
+  const eventImage = useMemo(
```

Comment:
Given that it's string, please remove useMemo

Reply:
you're preaching  to the choir 🙌 
https://github.com/sparkletown/sparkle/pull/2128/commits/8fc9e9becd300251bca5d4ca064d768167299581